modules/spring-config/src/main/resources/minimal-mule-config.xml

```diff
@@ -18,30 +16,57 @@
 
         <spring:bean name="_defaultPersistentQueueStore,_fileQueueStore" class="org.mule.util.store.DefaultObjectStoreFactoryBean" factory-method="createDefaultPersistentQueueStore"/>
 
-        <spring:bean name="_defaultInMemoryObjectStore" class="org.mule.util.store.DefaultObjectStoreFactoryBean" factory-method="createDefaultInMemoryObjectStore"/>
```

Comment:
Now that some beans were removed, can the minimal and the full config file share some code?